### PR TITLE
Booking Foreign Key Changes

### PR DESCRIPTION
**Pull request by akroon3r**
During a live client demo, it was determined that there needed to be changes made to the booking model and schema in order to accomplish the creation and listing of group exams in the booking feature. This PR addresses these requirements by making the room_id foreign key nullable, and adding an office_id foreign key that references the offices table. One additional PR will be required to update postman tests and swagger yaml files to communicate these changes to the client.

### CODE REVIEW

File: api/app/schemas/bookings/booking_schema.py [hunk 1]
```diff
@@ -32,6 +33,8 @@ class Meta:
     room_id = fields.Int()
     start_time = fields.DateTime()
     invigilator_id = fields.Int()
+    office_id = fields.Int()
 
     invigilator = fields.Nested(InvigilatorSchema())
     room = fields.Nested(RoomSchema(exclude=("booking", "office",)))
+    office = fields.Nested(OfficeSchema())
```
Comment (sjrumsby) [inline api/app/schemas/bookings/booking_schema.py]: Does ExamSchema contain a nested office in it as well? And/or does office contain nested bookings in it's schema? Want to make sure that we aren't including multiple things in here.

Comment (akroon3r) [reply api/app/schemas/bookings/booking_schema.py]: screen shot 2019-01-28 at 9 51 42 am

Comment (akroon3r) [reply api/app/schemas/bookings/booking_schema.py]: screen shot 2019-01-28 at 9 52 50 am

Comment (akroon3r) [reply api/app/schemas/bookings/booking_schema.py]: screen shot 2019-01-28 at 9 55 57 am

**Pull request merged**